.eslintrc.js

```diff
@@ -31,17 +20,15 @@ module.exports = {
 
   rules: {
     'no-console': 'off',
-    'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     '@typescript-eslint/no-explicit-any': 'off',
     '@typescript-eslint/ban-ts-ignore': 'off',
-    '@typescript-eslint/camelcase': 'off',
-    '@typescript-eslint/no-undef': 'off',
     '@typescript-eslint/no-var-requires': 'off',
-    '@typescript-eslint/no-use-before-define': 'off',
-    '@typescript-eslint/explicit-module-boundary-types': 'off',
     '@typescript-eslint/ban-ts-comment': 'off',
-    'simple-import-sort/imports': 'warn',
-    'simple-import-sort/exports': 'warn'
+    
+    // New rules
+    'vue/component-name-in-template-casing': ['error', 'PascalCase'],
+    'vue/multi-word-component-names': 'off'
```

Comment:
Vue recommends multi word component names, so they don't get mixed with regular html elements, but many components are already single word, so had to turn this off.